lib/committee/schema_validator/open_api_3/operation_wrapper.rb

```diff
@@ -89,9 +89,11 @@ def build_openapi_parser_path_option(validator_option)
 
         # @return [OpenAPIParser::SchemaValidator::Options]
         def build_openapi_parser_option(validator_option, coerce_value)
+          allow_empty_date_and_datetime = validator_option.allow_empty_date_and_datetime
           datetime_coerce_class = validator_option.coerce_date_times ? DateTime : nil
           validate_header = validator_option.check_header
-          OpenAPIParser::SchemaValidator::Options.new(coerce_value: coerce_value,
+          OpenAPIParser::SchemaValidator::Options.new(allow_empty_date_and_datetime: allow_empty_date_and_datetime,
```

Comment:
I think a good backwards compatible guard for this might be something like:
```
if OpenAPIParser::Config.instance_methods.include?(:allow_empty_date_and_datetime)
```

So if that is true, you know it recognizes the option and you pass it in. And if it isn't defined it would probably be good to output a warning that tells you it won't do anything and if you need/want it to that you would need to update versions or something.

What do you think?

Reply:
Good idea, I went with this:

```
if OpenAPIParser::SchemaValidator::Options.method_defined?(:allow_empty_date_and_datetime)
```

And then splat the keyword arguments.

I'm not sure if maybe using `method_defined?` is better because it allocates one less array (I didn't benchmark) but for us only using this in CI it doesn't matter. Happy to change it though if you have opinions.

Reply:
Yeah, `method_defined?` seems better. I was looking for that and didn't find it at the time, so grabbed something that would work (and hopefully get the point across). Thanks for finding the cleaner way.